definitions/npm/react-hot-loader_v4.6.x/flow_v0.53.0-/react-hot-loader_v4.6.x.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+declare type errorReporterProps = {|
+  error: Error,
+  errorInfo: { componentStack: string }
+|}
+
+declare type ContainerProps = {|
+  children: React$Element<any>,
+  errorBoundary?: boolean,
+  errorReporter?: React$ComponentType<errorReporterProps>,
+|}
```

Comment:
Those types should be scoped inside the module declaration.

Reply:
I guess `import type { ContainerProps } from 'react-hot-loader'` should work from `react-hot-loader/root` since they are on the same file (untested).

Reply:
> now they are global
I did not know.

Thank you for review. I'll fixed

Reply:
Thank you for review. I'll fixed

Reply:
I don't think `import`ing types across modules works, even if they are in the same file. I'm seeing a `Cannot import ContainerProps because there is no ContainerProps export in react-hot-loader.` error